darkmoon/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+
+<head>
```

Comment:
It looks like a bunch of stuff got duplicated on copy / paste into the template?

Reply:
Cleared out the duplicates!